nuxeo-core/nuxeo-core-api/src/test/java/org/nuxeo/ecm/core/api/model/impl/AbstractTestProperty.java

```diff
@@ -28,8 +29,15 @@
 import org.nuxeo.ecm.core.schema.types.SchemaImpl;
 import org.nuxeo.ecm.core.schema.types.Type;
 import org.nuxeo.ecm.core.schema.types.primitives.StringType;
+import org.nuxeo.runtime.test.NXRuntimeTestCase;
 
-public abstract class AbstractTestProperty {
+public abstract class AbstractTestProperty extends NXRuntimeTestCase {
```

Comment:
Don't use `NXRuntimeTestCase` (JUnit 3) but the `FeaturesRunner` and `RuntimeFeature` (JUnit 4) instead.
See https://doc.nuxeo.com/corg/unit-testing/